using-ide-vscode.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: Developing Flutter apps in an IDE
+
+permalink: /using-ide-vscode/
+---
+
+<ul class="tabs__top-bar">
+    <li class="tab-link" data-tab-href="/using-ide/">Android Studio / IntelliJ</li>
+    <li class="tab-link current">VS Code</li>
+</ul>
+
+<div class="tabs__content current" markdown="1">
+
+The Dart extension provides a fully integrated development experience in Visual Studio Code.
+
+* TOC Placeholder
+{:toc}
+
+## Installation and setup
+
+Please follow the [Editor Setup](/get-started/editor/) instructions to install
```

Comment:
Should this link to https://flutter.io/get-started/editor/#vscode ?

Reply:
I did that originally, but the link checker complains because there's no element with the ID `vscode` in that page. I couldn't think of an easy way to fix :(

Reply:
How do those link ids actually work? Looking at the generated jekyll output in `_site` I don't see `id` tags for them!?

Reply:
We use javascript to read the fragment from the url at page load, and then select the correct tab based on it. If we put real anchors in the page, it would try to scroll down which we don't want. What the link checker is doing seems reasonable, it's just not really valid for this case :(

Reply:
Got it. Maybe just mark those links to be ignored in the [`Rakefile`](https://github.com/flutter/website/blob/master/Rakefile)?

Reply:
Aha! I was looking around for whether I could configure it. Thanks!

Reply:
Actually looks like you're already skipping these so I've no idea why it was failing for me. I'll do some digging!

Reply:
Ok, seems like the exclusion we had may have been for the exact string "#" so I've changed it to /#/ which allows `#vscode` through, but it makes me think that all other hash fragments were previously being checked and the ignore rules wasn't really needed. Should I leave as being ignored, or see if I can make it more specific (only include #vscode) so it checks more links?

Also, seems like we have a load of 404s linking to developer.apple.com now? :(